tests/test_utils.py

```diff
@@ -176,7 +175,7 @@ def test_num_parameters_bitsandbytes(mode):
 
     with fabric.init_module(empty_init=True):
         model = GPT.from_name("pythia-14m")
-    assert num_parameters(model) == 14067712
+    assert num_parameters(model) == 13076352
```

Comment:
@awaelchli The previous number is the correct one. The calculation is wrong in some way

Reply:
The strange thing is that I cannot reproduce this issue on my machine

Reply:
@carmocca How did you calculate 14067712 to know it's right? 
The change in the number is probably related to us calling 
`p.quant_state.shape` now. I'll check what's the difference.

Reply:
I found that it's a test interaction. This 
```
pytest -v  tests/test_model.py::test_sdpa_choice tests/test_utils.py::test_num_parameters_bitsandbytes
```
is sufficient to repro. I'll follow the trail.

Reply:
Two tests were writing into the global `litgpt.config.configs` dict.

Reply:
I still don't understand how that makes the bnb test fail, but LGTM!

Reply:
Because the test was setting `config["n_layer"] = 1` globally, so every test that came after and using configs would use this. Reducing the number of layers would affect the param count, unrelated to BNB.